packages/server/integrations/mattermost/mattermostWebhookHandler.ts

```diff
@@ -0,0 +1,384 @@
+import {createVerifier, httpbis} from 'http-message-signatures'
+import {markdownToDraft} from 'markdown-draft-js'
+import {Variables} from 'relay-runtime'
+import {HttpRequest, HttpResponse} from 'uWebSockets.js'
+import AuthToken from '../../database/types/AuthToken'
+import uWSAsyncHandler from '../../graphql/uWSAsyncHandler'
+import parseBody from '../../parseBody'
+import getKysely from '../../postgres/getKysely'
+import getGraphQLExecutor from '../../utils/getGraphQLExecutor'
+import sendToSentry from '../../utils/sendToSentry'
+
+const MATTERMOST_SECRET = process.env.MATTERMOST_SECRET
+const PORT = __PRODUCTION__ ? process.env.PORT : process.env.SOCKET_PORT
+const HOST = process.env.HOST
+const PORT_SUFFIX = HOST !== 'localhost' ? '' : `:${PORT}`
+const ORIGIN = `${process.env.PROTO}://${HOST}${PORT_SUFFIX}`
+
+const markdownToDraftJS = (markdown: string) => {
+  const rawObject = markdownToDraft(markdown)
+  return JSON.stringify(rawObject)
+}
+
+const eventLookup: Record<
+  string,
+  {
+    query: string
+    convertResult?: (data: any) => any
+    convertInput?: (input: any) => any
+  }
+> = {
+  meetingTemplates: {
+    query: `
```

Comment:
+1 it'd be nice to validate these strings so if the schema changes we're alerted https://www.loom.com/share/effc307dce5b42e5992e36c7e7c9aee6?sid=bf22c73b-cac7-48ba-a110-d79bb0a3cdc3

we could add this eslint plugin to do the work for us: https://the-guild.dev/graphql/eslint/docs

note:

i had to update my graphql.config.js file to the following: 
```js
module.exports = {
  schema: './packages/server/graphql/public/schema.graphql',
  documents: './packages/server/**/*'
}
```

Reply:
Yes, that would be nice. I couldn't get the linting rule running in a reasonable amount of time, so I will add it in a separate PR.